### PR TITLE
chore: replace chalk with kleur

### PR DESCRIPTION
**Pull request by DanielRuf**
**Type:**
chore

The following has been addressed in the PR:

**Description:**

This PR replaces `chalk` with `kleur` which is smaller and faster.

`yarn add kleur && yarn remove chalk` to update the lockfile.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -5290,9 +5290,9 @@ kind-of@^6.0.0, kind-of@^6.0.2:
   resolved "https://registry.verdaccio.org/kind-of/-/kind-of-6.0.2.tgz#01146b36a6218e64e58f3a8d66de5d7fc6f6d051"
   integrity sha512-s5kLOcnH0XqDO+FvuaLX8DDjZ18CGFk7VygH40QoKPUQhW4e2rvM0rwUq0t8IQDOwYSeLK01U90OjzBTme2QqA==
 
-kleur@^3.0.2:
+kleur@3.0.3, kleur@^3.0.2:
   version "3.0.3"
-  resolved "https://registry.verdaccio.org/kleur/-/kleur-3.0.3.tgz#a79c9ecc86ee1ce3fa6206d1216c501f147fc07e"
+  resolved "https://registry.npmjs.org/kleur/-/kleur-3.0.3.tgz#a79c9ecc86ee1ce3fa6206d1216c501f147fc07e"
```
Comment (DanielRuf) [inline yarn.lock]: @juanpicado is the `.npmrc` file in the repo correct? It uses the npm registry.

Comment (juanpicado) [reply yarn.lock]: :-) my bad ... I use `nrm` command often.

Comment (juanpicado) [reply yarn.lock]: ahh, well, the lock file points to our registry in purpose. So, we do not have to deal with registry timeouts.

**Comment by juanpicado** [comment]
Unfortunately, we need to take care of flow here .... you need to create the types, since we are going to remove Flow soon, just create a stub. 

```
➜ flow-typed create-stub kleur@3.0.3             
• Creating 1 stub...
  • kleur@3.0.3
    └> flow-typed/npm/kleur_vx.x.x.js
```
or just create `flow-typed/npm/kleur_vx.x.x.js`

```
// flow-typed signature: 27250091946c7286334b6d8e332b126f
// flow-typed version: <<STUB>>/kleur_v3.0.3/flow_v0.81.0

/**
 * This is an autogenerated libdef stub for:
 *
 *   'kleur'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module 'kleur' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module 'kleur/index' {
  declare module.exports: $Exports<'kleur'>;
}
declare module 'kleur/index.js' {
  declare module.exports: $Exports<'kleur'>;
}
```

**Comment by juanpicado** [comment]
I did not know `chalk` was so slow, nice stuff https://github.com/lukeed/kleur#performance thanks @DanielRuf

**Comment by DanielRuf** [comment]
Thanks for the review. Is it the same for the `dayjs` PR?

**Comment by juanpicado** [comment]
For some reason `dayjs` was fine. 🤔  ...

**Comment by juanpicado** [comment]
Ahh, because logger is not tagged as `@flow` file. 💡

**Comment by DanielRuf** [comment]
> Ahh, because logger is not tagged as `@flow` file. 💡

Ah, this makes sense.

**Comment by DanielRuf** [comment]
I've pushed 3 new commits which should resolve the findings.

**Comment by DanielRuf** [comment]
Seems I forgot logger.js, I will push another commit to migrate this one too.

File: src/lib/cli.js [hunk 2]
```diff
@@ -8,20 +8,20 @@
 
 import path from 'path';
 import semver from 'semver';
-import chalk from 'chalk';
+const { bgYellow, bgRed } = require('kleur');
```
Comment (juanpicado) [inline src/lib/cli.js]: I'd preffer here `import 'kleur';` .. unless they do not follow ES modules.

Comment (juanpicado) [reply src/lib/cli.js]: Same in other places.

Comment (DanielRuf) [reply src/lib/cli.js]: Ok, I'll change it.

Comment (juanpicado) [reply src/lib/cli.js]: The reason is https://github.com/verdaccio/eslint-config-verdaccio is more aggressive with linting, after migrating to Typescript it only accepts import (and only require in some specific scenarios)

**Comment by juanpicado** [comment]
Thanks @DanielRuf works pretty good :-) this will improve performance for sure.

**Pull request merged**